sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -278,6 +274,45 @@ static status_t config_and_erase_certificate_flash_pages(void) {
   return OK_STATUS();
 }
 
+/**
+ * Erase all of the owner's INFO pages so that they're in a known state.
+ */
+static status_t erase_owner_info_pages(owner_config_t *config) {
+  const flash_ctrl_info_page_t *pages[] = {
+      &kFlashCtrlInfoPageOwnerReserved0, &kFlashCtrlInfoPageOwnerReserved1,
+      &kFlashCtrlInfoPageOwnerReserved2, &kFlashCtrlInfoPageOwnerReserved3,
+      &kFlashCtrlInfoPageOwnerReserved4, &kFlashCtrlInfoPageOwnerReserved5,
+      &kFlashCtrlInfoPageOwnerReserved6, &kFlashCtrlInfoPageOwnerReserved7,
+  };
+
+  // First, initialize all of the owner INFO pages with ECC & Scrambling.
+  for (size_t i = 0; i < ARRAYSIZE(pages); ++i) {
+    flash_ctrl_cfg_t cfg = {
+        .scrambling = kMultiBitBool4True,
```

Comment:
Shouldn't we use the owner info configuration for scrambling bit here before we do the erase? If the owner disabled scrambling won't the page be in a bad state?

Reply:
This is to set defaults, and later the `TRY(owner_block_info_apply(config->info));` call sets the settings included in the owner config.

Reply:
Gotcha, thank!